cdap-ui/app/cdap/components/AbstractWidget/ToggleSwitchWidget/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { objectQuery } from 'services/helpers';
+
+import ToggleSwitch from 'components/ToggleSwitch';
+
+interface IConfigType {
+  'widget-attributes': IWidgetAttributes;
+}
+interface IToggle {
+  label: string;
+  value: string;
+}
+interface IWidgetAttributes {
+  on: IToggle;
+  off: IToggle;
+  default: string;
```

Comment:
Do we know if `default` will always be provided? Can we make it optional in implementation and fallback to `on` if provided?